src/js/timer.js

```diff
@@ -0,0 +1,22 @@
+import { removeAllSpinners } from './utils.js';
+import { DELAY } from './constants.js';
+
+const getIntervalTimer = (createTemplate, calculate, cars, carPlayers, coin) => {
```

Comment:
```suggestion
const getIntervalTimer = ({ createTemplate, calculate, cars, carPlayers, coin }) => {
```

그리고 함수의 인자가 많아지는 경우에는 객체로 묶어서 받아주시는게 어떨까요?

Reply:
이부분은 잘 이해하지 못했습니다 ㅠ

Reply:
```js
// 선언
const getIntervalTimer = (createTemplate, calculate, cars, carPlayers, coin) => {
...
}

// 사용
getIntervalTimer(createTemplate, calculate, cars, carPlayers, coin) // O
getIntervalTimer(createTemplate, calculate, coin, cars, carPlayers) // X
```

현재 위와 같은 형식으로 코드가 선언되어 있다면 5개의 인자를 넘길 때 항상 순서를 맞춰서 넣어줘야 하는데요. 이는 개발자가 parameter의 순서를 모두 알고, 기억해야 된다는 의미이기도 합니다.

하지만 아래와 같이 객체를 통해 인자를 받게 된다면 parameter의 순서와 관계없이 값을 받아서 사용할 수 있어서 해당 함수를 사용할 때의 실수를 줄여줄 수 있을거 같네용

```js
// 선언
const getIntervalTimer = ({ createTemplate, calculate, cars, carPlayers, coin }) => {
...
}

// 사용
getIntervalTimer({ createTemplate, calculate, cars, carPlayers, coin }) // O
getIntervalTimer({ createTemplate, calculate, coin, cars, carPlayers }) // O

Reply:
우와 이런게 있었군요!! 좋은정보 얻어갑니다 감사합니다!! 
> 객체를 통해 인자를 받게 된다면 parameter의 순서와 관계없이 값을 받아서 사용할 수 있어서 해당 함수를 사용할수 있다.

모르던정보였네요 감사합니다~!